api/test/metabolites.test.js

```diff
@@ -0,0 +1,21 @@
+import fetch from "node-fetch";
+
+describe("metabolites", () => {
+  test("a metabolite should have a list of subsystems", async () => {
+    const res = await fetch(
+      `${API_BASE}/metabolites/MAM01199m?model=HumanGem&version=${HUMAN_GEM_VERSION}`
+    );
+
+    const { subsystems } = await res.json();
+    expect(subsystems.length).toBeGreaterThan(0);
```

Comment:
Check that all sybsystems have `id` and `name`? Is there any specific subsystem for `MAM01199m` that must be present?

Reply:
I think a unit test might be suitable as mentioned in [this comment](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/830#issuecomment-1019984656).